blog.html

```diff
@@ -0,0 +1,268 @@
+                <li>List name <span>&nbsp</span></li>
+                <li>List name <span>&nbsp</span></li>
+                <li>List name <span>&nbsp</span></li>
+            </ol>
+            <div class="blog__blockquote">
+                <blockquote cite="https://">
+                    <p class="blog__blockquote--text">“The first huge metaverse arena to arrange live sports, hang out and even make bets and wagers”</p>
+                </blockquote>
+                <p class="blog__blockquote--rating"><span class="blog__blockquote--span">8.5/10</span><cite>Markout Corporation</cite></p>
+            </div>
+            <div class="blog-content__photo-wrapper">
+                <figure>
+                    <img src="images/dragon-baby.jpg" width="448" height="390" alt="dragon baby">
+                    <!-- <figcaption class="blog__photo-caption">Photo caption</figcaption> -->
+                </figure>
+                <figure>
+                    <img src="images/dragon-baby-and-man.jpg" width="448" height="390" alt="dragon baby and man">
+                    <!-- <figcaption class="blog__photo-caption">Photo caption</figcaption> -->
+                </figure>
+            </div>
+            <h3 class="intro__subtitle">NFTs are digital assets that represent real-world objects</h3>
+            <p>Setting up your avatar and personalizing your nickname on Binance is quick and easy. Users can go to binance.com under 
+                Dashboard > Settings > My Profile > Avatar and Nickname. For those of you who own NFT assets, you can select the NFT you
+                want to use from your collection as your avatar to personalize your profile. For users who do not own any NFT avatars, 
+                check out the NFT avatar collections page to get started.</p>
+
+                <p>Alternatively, if you’re on nft.binance.com, go to User Center > Settings > Basic > Set Avatar and Nickname, and you’ll be 
+                redirected to the main site. Binance NFT Creators have the option of setting their Avatars on nft.binance.com. Once you set 
+                your new unique nickname, it will be standardized and used across the Binance ecosystem.</p>
+
+            <h3 class="intro__subtitle">NFT Avatar Generators</h3>
+            <p>You can find a range of NFT marketplaces where you can buy existing NFTs or list your own NFTs for sale.</p>
+            <footer class="blog-footer">
+                <div class="blog-footer__wrapper">
+                    <span>Tags:</span>
+                    <ul class="tags__wrapper">
+                        <li class="tags__item"><a href="#">NFT</a></li>
+                        <li class="tags__item"><a href="#">ICO</a></li>
+                        <li class="tags__item"><a href="#">Crypto</a></li>
+                    </ul>
+                </div>
+                <div class="blog-footer__wrapper">
+                    <span>Share:</span>
+                    <ul class="social">
```

Comment:
There is should be social-alt list with smaller icons and border around the links.

Reply:
done via modifier